app/components/ApplicationPage/AuthPage/ResetPasswordPage/index.js

```diff
@@ -2,19 +2,31 @@
 
 import * as React from 'react';
 import { translate, type TranslatorProps } from 'react-i18next';
+import { type ContextRouter as RouterProps } from 'react-router-dom';
 
-import ContainerPageWrapper from 'components/ContainerPageWrapper';
+import { InvalidArgumentError } from 'errors';
+import PageWrapper from 'components/PageWrapper';
```

Comment:
You actually do need to use `ContainerPageWrapper` instead of `PageWrapper` if you want the flash messages to be shown, since a plain `PageWrapper` only adds the navbar and doesn't contain any other functionality. Same for `ConfirmEmailPage`.